docs/user/cli/ingest.rst

```diff
@@ -90,7 +88,8 @@ and Microsoft's Azure file systems are supported with a few configuration change
 :doc:`/user/cli/filesystems` for details.
 
 Instead of specifying files, input data may be piped directly to the ingest command using `stdin` shell redirection.
-Note that this will only work in local mode, and will only use a single thread for ingestion. Progress indicators
-may not be entirely accurate, as the total size isn't known up front. For example::
+Note that this will only work in local mode, and will only use a single thread for ingestion. Schema inference is
+disabled in this case, and progress indicators may not be entirely accurate, as the total size isn't known up front.
+For example::
```

Comment:
Is it worth being pedantic and suggesting that users with lots of files can 1) do schema inference on a sample file, 2) persist the config, and 3) run their larger ingest?

Reply:
it's always worth being pedantic ;)